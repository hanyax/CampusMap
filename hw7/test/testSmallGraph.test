# Load the test small graph and find the paths for:

LoadGraph test smallLeastCostGraph.tsv

# Most Apart:
FindPath test D H

# Adjacent:
FindPath test A B

# Middle Distance:
FindPath test A E
