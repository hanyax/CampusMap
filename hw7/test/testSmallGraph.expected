# Load the test small graph and find the paths for:

loaded graph test

# Most Apart:
path from D to H:
D to E with weight 0.200
E to B with weight 0.125
B to F with weight 0.500
F to H with weight 0.333
total cost: 1.158

# Adjacent:
path from A to B:
A to B with weight 0.500
total cost: 0.500

# Middle Distance:
path from A to E:
A to D with weight 0.250
D to E with weight 0.200
total cost: 0.450
